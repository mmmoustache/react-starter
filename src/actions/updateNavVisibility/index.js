const updateNavVisibility = input => ({
  type: 'UPDATE_NAV_VISIBILITY',
  text: input || input,
});

export default updateNavVisibility;
