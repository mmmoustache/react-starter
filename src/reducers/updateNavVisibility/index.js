export default function (state = null, action) {
  switch (action.type) {
    case 'UPDATE_NAV_VISIBILITY':
      return action.text ? action.text : null;
      break;
    default:
      break;
  }
  return state;
}
