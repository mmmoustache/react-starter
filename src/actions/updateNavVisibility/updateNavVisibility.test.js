import updateNavVisibility from '../updateNavVisibility';

describe('Action Creator: updateNavVisibility', () => {
  it('should match its snapshot', () => {
    const result = updateNavVisibility();
    expect(result).toMatchSnapshot();
  })
});