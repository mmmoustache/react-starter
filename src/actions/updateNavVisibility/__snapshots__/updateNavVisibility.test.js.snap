// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Action Creator: updateNavVisibility should match its snapshot 1`] = `
Object {
  "text": undefined,
  "type": "UPDATE_NAV_VISIBILITY",
}
`;
